jumpscale/packages/vdc_dashboard/bottle/api/deployments.py

```diff
@@ -247,6 +247,38 @@ def threebot_vdc():
     )
 
 
+@app.route("/api/vdc/expiration", method="GET")
+@package_authorized("vdc_dashboard")
+def vdc_expiration():
+    vdc = get_vdc()
```

Comment:
`vdc` is not used here? or do we need to call `get_vdc` to fetch information that `get_expiration_data` depends on?

Reply:
No, we don't need it. the `require_vdc_info` wrapper function responsible for getting the VDC info